server/embed/config_test.go

```diff
@@ -72,6 +76,14 @@ func TestConfigFileOtherFields(t *testing.T) {
 		t.Errorf("ForceNewCluster = %v, want %v", cfg.ForceNewCluster, true)
 	}
 
+	if cfg.SocketOpts.ReusePort == false {
```

Comment:
Optional: 
I found 
```
assert.Equals(t, true, cfg.SocketOpts.ReusePort, "ReusePort does not match") 
```
shorter and providing a nice output in case of failure.